pkg/image/trigger/annotations/annotations.go

```diff
@@ -172,6 +172,33 @@ func UpdateObjectFromImages(obj runtime.Object, tagRetriever trigger.TagRetrieve
 	return updated, nil
 }
 
+// ContainerImageChanged returns true if any container image referenced by newTriggers changed.
+func ContainerImageChanged(oldObj, newObj runtime.Object, newTriggers []triggerapi.ObjectFieldTrigger) bool {
+	for _, trigger := range newTriggers {
+		if trigger.Paused {
+			continue
+		}
+
+		newContainer, _, err := ContainerForObjectFieldPath(newObj, trigger.FieldPath)
+		if err != nil {
+			glog.V(5).Infof("%v", err)
+			continue
+		}
+
+		oldContainer, _, err := ContainerForObjectFieldPath(oldObj, trigger.FieldPath)
+		if err != nil {
+			// might just be a result of the update
```

Comment:
glog(5) here as well

Reply:
if the triggers has changed or the images and the field path is valid only for the new image+trigger combination there will always be an error here although it's not an error in this case